### ISSUE TITLE
How to upgrade mondrian version 4.x from 3.x

### ISSUE THREAD

**Issue by foryou2030** [pentaho/mondrian#767]
I install  mondrian with version 3.5, ann connect it to hive, it can work now.
But once I upgrade it to 4.3, it can't worn any way.
 update step:
 download mondrian-4.3.jar, then replace the mondrian-3.5.jar with the new jar.

after upgraded, it  throw some exception:
ex1:  java.lang.NoClassDefFoundError: mondrian/xmla/XmlaHandler$ConnectionFactory (resolved)
ex2: java.lang.NoSuchMethodError: org.apache.log4j.Logger.isTraceEnabled()Z mondrian
and i think i will face another issues even though  i resloved above.

Anybody  can give me an upgrade guide?

**Comment by Zhangshunyu** [comment]
Yes, could any one pls tell that how to install Mondrian 4.x after 3.x is installed, since there is no .war package yet.
